src/media/js/views/search.js

```diff
@@ -208,9 +208,16 @@ define('views/search',
             if (query) break;
         }
 
-        if (query && queryParam === 'author') {
-            pageTypes += ' leaf';
-            utilsLocal.headerTitle(gettext('Developer Listing'));
+        if (query) {
+            switch (queryParam) {
+                case 'author':
+                    pageTypes += ' leaf';
```

Comment:
would be cool if you changed pageTypes to be an array, and then did an Array.join below